src/BiomeJsBinary.php

```diff
@@ -121,6 +123,13 @@ private function getLatestVersion(): string
     {
         $useStable = null === $this->binaryVersion || 'latest_stable' === $this->binaryVersion;
         $useNightly = 'latest_nightly' === $this->binaryVersion;
+        $cacheItem = $this->cache->getItem('latest_version_' . ($useStable ? 'use_stable' : 'dont_use_stable') . '_' . ($useNightly ? 'use_nightly' : 'dont_use_nightly'));
```

Comment:
The cache key creation can be simplified IMO, also I want it to depends on the current OS/architecture (with `getBinaryName`):
```suggestion
        $cacheKey = sprintf(
            'binary.latest_version.%s.%s',
            match(true) {
                $useStable => 'stable',
                $useNightly => 'nightly',
                default => throw new \LogicException('Invalid configuration'),
                },
            self::getBinaryName();    
        );
        $cacheItem = $this->cache->getItem($cacheKey);
```